deploy/monitor/push_service_monitor.yaml

```diff
@@ -7,8 +7,9 @@ metadata:
   name: unifiedpush
 spec:
   endpoints:
-    - path: /api/metrics
-      port: server
+    - path: /rest/prometheus/metrics
+      port: web
   selector:
     matchLabels:
-      app: unifiedpush
\ No newline at end of file
+      service:
```

Comment:
we don't need this line. I don't think it's valid.

Reply:
Yep, it was the cause of issue.